e2e/cypress/integration/bot_accounts/bot_api_spec.js

```diff
@@ -7,6 +7,7 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
+// Stage: @prod
```

Comment:
Is this intended?

Reply:
No, and not sure where it would have come from. I reverted these 2 files to be the same as master. All good.

Reply:
Thanks!